packages/http-client-csharp/emitter/src/sdk-context.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+import { SdkContext, SdkType } from "@azure-tools/typespec-client-generator-core";
+import { Logger } from "./lib/logger.js";
+import { CSharpEmitterOptions } from "./options.js";
+import { InputEnumType, InputModelType, InputType } from "./type/input-type.js";
+
+export interface CSharpEmitterContext extends SdkContext<CSharpEmitterOptions> {
+  logger: Logger;
+  __typeCache: SdkTypeMap;
```

Comment:
Why the double underscore? Shouldn't it be single underscore if it is used internally?

Reply:
Nevermind, I guess it needs to be double since we export this.